Sports.py

```diff
@@ -2,49 +2,248 @@
 #
 # Title:      Sports Display 64x128 LED Matrix
 # Author:     Timothy Kosinski
-# Date:       06MAR2024
+# Date:       12APR2024
 # Description:
 #  Drives my LED Matrix Panel, Adifruit Display
+# Takes In NBA Player Names and Displays Players Stats
+# By using api_calls.py (API) to run the display
 #==========================================================
 
 #!/usr/bin/env python
 from samplebase import SampleBase
 from rgbmatrix import graphics
 from datetime import datetime
+from PIL import Image, ImageOps, ImageEnhance 
+
+from api_calls import GameFacade
+import api_calls
+import http.client
 import time
+import socket
+import os

```

Comment:
these could be repeated from API imports, could be condensed to one line!

Reply:
will do